developer/src/common/include/kmn_compiler_errors.h

```diff
@@ -184,7 +184,7 @@
 #define CWARN_ANSIInUnicodeGroup                           0x00002087
 #define CWARN_UnicodeSurrogateUsed                         0x00002088
 #define CWARN_ReservedCharacter                            0x00002089
-#define CWARN_Info                                         0x0000208A
+#define CWARN_Info                                         0x0000008A
```

Comment:
A comment similar to the one in `developer/src/kmc-kmn/src/compiler/messages.ts` might be helpful to explain why the value is not in the 0x2000 range.

Also it might be helpful to additionally define `CINFO_Info`:

```suggestion
#define CWARN_Info                                         0x0000008A
#define CINFO_Info                                         CWARN_Info
```